### DISCUSSION TITLE
Synology LDAP Server | Error 500

### DISCUSSION THREAD

**Discussion by stormframe** [kimai/kimai#4610] [General] [answered]
Hello.
we are getting an Error 500 trying to connect to the LDAP.

Ubuntu 22.04 Installaion

local.yaml

```
kimai:
     ldap:
         activate: true
         connection:
             host: "ldap://192.168.200.210"
             port: 389
             username: uid=root,cn=users,dc=ldap,dc=firemanager,dc=local
             password: ***********
             accountFilterFormat: (&(objectClass=person)(sambaSamAccount=%s))
             accountDomainName: firemanager.local
             accountDomainNameShort: local
         user:
             baseDn: DC=firemanager,DC=local
             usernameAttribute: uid
             filter: (&(objectClass=person))
             attributesFilter: (objectClass=person)
             attributes:
                 - { ldap_attr: samaccountname,  user_method: setUserIdentifier }
                 - { ldap_attr: mail, user_method: setEmail }
                 - { ldap_attr: displayname, user_method: setAlias }
         role:
             baseDn: DC=firemanager,DC=local
             filter: (&(objectClass=group))
             groups:
                 - { ldap_value: Domain Admins, role: ROLE_SUPER_ADMIN }
                 - { ldap_value: zeitmanager, role: ROLE_ADMIN }
                 - { ldap_value: Users, role: ROLE_USER }
```
Exception:

``` 
           // searchEntries don't return 'count' key as specified by php native function ldap_get_entries()            $entries['count'] = \count($entries);        } catch (LdapException $exception) {            $this->ldapExceptionHandler($exception);            throw new LdapDriverException('An error occurred with the search operation.');        }        return $entries;    }

```

Stack:
`App\Ldap\LdapDriverException:
An error occurred with the search operation.

  at src/Ldap/LdapDriver.php:72
  at App\Ldap\LdapDriver->search()
     (src/Ldap/LdapManager.php:41)
  at App\Ldap\LdapManager->findUserByUsername()
     (src/Ldap/LdapUserProvider.php:30)
  at App\Ldap\LdapUserProvider->loadUserByIdentifier()
     (vendor/symfony/security-core/User/ChainUserProvider.php:63)
  at Symfony\Component\Security\Core\User\ChainUserProvider->loadUserByIdentifier()
     (src/Security/KimaiUserProvider.php:64)
  at App\Security\KimaiUserProvider->loadUserByIdentifier()
     (vendor/symfony/security-http/Authenticator/Passport/Badge/UserBadge.php:87)
  at Symfony\Component\Security\Http\Authenticator\Passport\Badge\UserBadge->getUser()
     (vendor/symfony/security-http/Authenticator/Passport/Passport.php:56)
  at Symfony\Component\Security\Http\Authenticator\Passport\Passport->getUser()
     (vendor/symfony/security-http/EventListener/UserCheckerListener.php:42)
  at Symfony\Component\Security\Http\EventListener\UserCheckerListener->preCheckCredentials()
     (vendor/symfony/event-dispatcher/Debug/WrappedListener.php:116)
  at Symfony\Component\EventDispatcher\Debug\WrappedListener->__invoke()
     (vendor/symfony/event-dispatcher/EventDispatcher.php:220)
  at Symfony\Component\EventDispatcher\EventDispatcher->callListeners()
     (vendor/symfony/event-dispatcher/EventDispatcher.php:56)
  at Symfony\Component\EventDispatcher\EventDispatcher->dispatch()
     (vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php:139)
  at Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch()
     (vendor/symfony/security-http/Authentication/AuthenticatorManager.php:180)
  at Symfony\Component\Security\Http\Authentication\AuthenticatorManager->executeAuthenticator()
     (vendor/symfony/security-http/Authentication/AuthenticatorManager.php:158)
  at Symfony\Component\Security\Http\Authentication\AuthenticatorManager->executeAuthenticators()
     (vendor/symfony/security-http/Authentication/AuthenticatorManager.php:140)
  at Symfony\Component\Security\Http\Authentication\AuthenticatorManager->authenticateRequest()
     (vendor/symfony/security-http/Firewall/AuthenticatorManagerListener.php:40)
  at Symfony\Component\Security\Http\Firewall\AuthenticatorManagerListener->authenticate()
     (vendor/symfony/security-http/Authenticator/Debug/TraceableAuthenticatorManagerListener.php:68)
  at Symfony\Component\Security\Http\Authenticator\Debug\TraceableAuthenticatorManagerListener->authenticate()
     (vendor/symfony/security-bundle/Debug/WrappedLazyListener.php:46)
  at Symfony\Bundle\SecurityBundle\Debug\WrappedLazyListener->authenticate()
     (vendor/symfony/security-http/Firewall/AbstractListener.php:26)
  at Symfony\Component\Security\Http\Firewall\AbstractListener->__invoke()
     (vendor/symfony/security-bundle/Debug/TraceableFirewallListener.php:83)
  at Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener->callListeners()
     (vendor/symfony/security-http/Firewall.php:95)
  at Symfony\Component\Security\Http\Firewall->onKernelRequest()
     (vendor/symfony/event-dispatcher/Debug/WrappedListener.php:116)
  at Symfony\Component\EventDispatcher\Debug\WrappedListener->__invoke()
     (vendor/symfony/event-dispatcher/EventDispatcher.php:220)
  at Symfony\Component\EventDispatcher\EventDispatcher->callListeners()
     (vendor/symfony/event-dispatcher/EventDispatcher.php:56)
  at Symfony\Component\EventDispatcher\EventDispatcher->dispatch()
     (vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php:139)
  at Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher->dispatch()
     (vendor/symfony/http-kernel/HttpKernel.php:157)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw()
     (vendor/symfony/http-kernel/HttpKernel.php:76)
  at Symfony\Component\HttpKernel\HttpKernel->handle()
     (vendor/symfony/http-kernel/Kernel.php:197)
  at Symfony\Component\HttpKernel\Kernel->handle()
     (vendor/symfony/runtime/Runner/Symfony/HttpKernelRunner.php:35)
  at Symfony\Component\Runtime\Runner\Symfony\HttpKernelRunner->run()
     (vendor/autoload_runtime.php:29)
  at require_once('/var/www/kimai/vendor/autoload_runtime.php')
     (public/index.php:5)                `

Log:
`<html>
<body>

DEBUG19:00:43 | app | ldap_search(DC=firemanager,DC=local, (&(&(objectClass=person))(uid=Michael)), {attributes}){     "action": "ldap_search",     "base_dn": "DC=firemanager,DC=local",     "filter": "(&(&(objectClass=person))(uid=Michael))",     "attributes": [         "+",         "*"     ] }
-- | -- | --

</body>
</html>`

`<html>
<body>

INFO19:00:43 | request | Matched route "security_check".{     "route": "security_check",     "route_parameters": {         "_route": "security_check",         "_controller": "App\\Controller\\Security\\SecurityController::checkAction",         "_locale": "de"     },     "request_uri": "http://192.168.200.8/de/login_check",     "method": "POST" }
-- | -- | --
INFO19:00:43 | doctrine | Connecting with parameters {params}{     "params": {         "driver": "pdo_mysql",         "charset": "utf8mb4",         "host": "127.0.0.1",         "port": 3306,         "user": "kimai",         "password": "<redacted>",         "driverOptions": [],         "defaultTableOptions": {             "charset": "utf8mb4",             "collation": "utf8mb4_unicode_ci"         },         "dbname": "kimai",         "serverVersion": "10.5.8-MariaDB"     } }
DEBUG19:00:43 | security | Checking for authenticator support.{     "firewall_name": "secured_area",     "authenticators": 7 }
DEBUG19:00:43 | security | Checking support on authenticator.{     "firewall_name": "secured_area",     "authenticator": "App\\API\\Authentication\\SessionAuthenticator" }
DEBUG19:00:43 | security | Authenticator does not support the request.{     "firewall_name": "secured_area",     "authenticator": "App\\API\\Authentication\\SessionAuthenticator" }
DEBUG19:00:43 | security | Checking support on authenticator.{     "firewall_name": "secured_area",     "authenticator": "App\\Saml\\SamlAuthenticator" }
DEBUG19:00:43 | security | Authenticator does not support the request.{     "firewall_name": "secured_area",     "authenticator": "App\\Saml\\SamlAuthenticator" }
DEBUG19:00:43 | security | Checking support on authenticator.{     "firewall_name": "secured_area",     "authenticator": "Scheb\\TwoFactorBundle\\Security\\Http\\Authenticator\\TwoFactorAuthenticator" }
DEBUG19:00:43 | security | Authenticator does not support the request.{     "firewall_name": "secured_area",     "authenticator": "Scheb\\TwoFactorBundle\\Security\\Http\\Authenticator\\TwoFactorAuthenticator" }
DEBUG19:00:43 | security | Checking support on authenticator.{     "firewall_name": "secured_area",     "authenticator": "Symfony\\Component\\Security\\Http\\Authenticator\\LoginLinkAuthenticator" }
DEBUG19:00:43 | security | Authenticator does not support the request.{     "firewall_name": "secured_area",     "authenticator": "Symfony\\Component\\Security\\Http\\Authenticator\\LoginLinkAuthenticator" }
DEBUG19:00:43 | security | Checking support on authenticator.{     "firewall_name": "secured_area",     "authenticator": "App\\Ldap\\LdapAuthenticator" }
DEBUG19:00:43 | security | Checking support on authenticator.{     "firewall_name": "secured_area",     "authenticator": "Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator" }
DEBUG19:00:43 | security | Checking support on authenticator.{     "firewall_name": "secured_area",     "authenticator": "Symfony\\Component\\Security\\Http\\Authenticator\\RememberMeAuthenticator" }
DEBUG19:00:43 | security | Authenticator does not support the request.{     "firewall_name": "secured_area",     "authenticator": "Symfony\\Component\\Security\\Http\\Authenticator\\RememberMeAuthenticator" }
DEBUG19:00:43 | doctrine | Executing statement: SELECT k0_.id AS id_0, k0_.alias AS alias_1, k0_.registration_date AS registration_date_2, k0_.title AS title_3, k0_.avatar AS avatar_4, k0_.api_token AS api_token_5, k0_.auth AS auth_6, k0_.username AS username_7, k0_.email AS email_8, k0_.account AS account_9, k0_.enabled AS enabled_10, k0_.password AS password_11, k0_.last_login AS last_login_12, k0_.confirmation_token AS confirmation_token_13, k0_.password_requested_at AS password_requested_at_14, k0_.roles AS roles_15, k0_.totp_secret AS totp_secret_16, k0_.totp_enabled AS totp_enabled_17, k0_.system_account AS system_account_18, k0_.color AS color_19, k0_.supervisor_id AS supervisor_id_20 FROM kimai2_users k0_ WHERE k0_.username = ? OR k0_.email = ? (parameters: {params}, types: {types}){     "sql": "SELECT k0_.id AS id_0, k0_.alias AS alias_1, k0_.registration_date AS registration_date_2, k0_.title AS title_3, k0_.avatar AS avatar_4, k0_.api_token AS api_token_5, k0_.auth AS auth_6, k0_.username AS username_7, k0_.email AS email_8, k0_.account AS account_9, k0_.enabled AS enabled_10, k0_.password AS password_11, k0_.last_login AS last_login_12, k0_.confirmation_token AS confirmation_token_13, k0_.password_requested_at AS password_requested_at_14, k0_.roles AS roles_15, k0_.totp_secret AS totp_secret_16, k0_.totp_enabled AS totp_enabled_17, k0_.system_account AS system_account_18, k0_.color AS color_19, k0_.supervisor_id AS supervisor_id_20 FROM kimai2_users k0_ WHERE k0_.username = ? OR k0_.email = ?",     "params": {         "1": "Michael",         "2": "Michael"     },     "types": {         "1": 2,         "2": 2     } }
DEBUG19:00:43 | app | ldap_search(DC=firemanager,DC=local, (&(&(objectClass=person))(uid=Michael)), {attributes}){     "action": "ldap_search",     "base_dn": "DC=firemanager,DC=local",     "filter": "(&(&(objectClass=person))(uid=Michael))",     "attributes": [         "+",         "*"     ] }
DEBUG19:00:43 | app | {exception}{     "exception": {} }
DEBUG19:00:43 | app | Notified event "Symfony\Component\Security\Http\Event\CheckPassportEvent" to listener "Symfony\Component\Security\Http\EventListener\LoginThrottlingListener::checkPassport".{     "event": "Symfony\\Component\\Security\\Http\\Event\\CheckPassportEvent",     "listener": "Symfony\\Component\\Security\\Http\\EventListener\\LoginThrottlingListener::checkPassport" }
DEBUG19:00:43 | app | Notified event "Symfony\Component\Security\Http\Event\CheckPassportEvent" to listener "Symfony\Component\Security\Http\EventListener\UserProviderListener::checkPassport".{     "event": "Symfony\\Component\\Security\\Http\\Event\\CheckPassportEvent",     "listener": "Symfony\\Component\\Security\\Http\\EventListener\\UserProviderListener::checkPassport" }
DEBUG19:00:43 | app | Notified event "Symfony\Component\Security\Http\Event\CheckPassportEvent" to listener "Symfony\Component\Security\Http\EventListener\UserProviderListener::checkPassport".{     "event": "Symfony\\Component\\Security\\Http\\Event\\CheckPassportEvent",     "listener": "Symfony\\Component\\Security\\Http\\EventListener\\UserProviderListener::checkPassport" }
DEBUG19:00:43 | app | Notified event "Symfony\Component\Security\Http\Event\CheckPassportEvent" to listener "Symfony\Component\Security\Http\EventListener\CsrfProtectionListener::checkPassport".{     "event": "Symfony\\Component\\Security\\Http\\Event\\CheckPassportEvent",     "listener": "Symfony\\Component\\Security\\Http\\EventListener\\CsrfProtectionListener::checkPassport" }
DEBUG19:00:43 | app | Notified event "Symfony\Component\Security\Http\Event\CheckPassportEvent" to listener "Symfony\Component\Security\Http\EventListener\UserCheckerListener::preCheckCredentials".{     "event": "Symfony\\Component\\Security\\Http\\Event\\CheckPassportEvent",     "listener": "Symfony\\Component\\Security\\Http\\EventListener\\UserCheckerListener::preCheckCredentials" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\DebugHandlersListener::configure".{     "event": "kernel.request",     "listener": "Symfony\\Component\\HttpKernel\\EventListener\\DebugHandlersListener::configure" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\ValidateRequestListener::onKernelRequest".{     "event": "kernel.request",     "listener": "Symfony\\Component\\HttpKernel\\EventListener\\ValidateRequestListener::onKernelRequest" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "Nelmio\CorsBundle\EventListener\CorsListener::onKernelRequest".{     "event": "kernel.request",     "listener": "Nelmio\\CorsBundle\\EventListener\\CorsListener::onKernelRequest" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "FOS\RestBundle\EventListener\ZoneMatcherListener::onKernelRequest".{     "event": "kernel.request",     "listener": "FOS\\RestBundle\\EventListener\\ZoneMatcherListener::onKernelRequest" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\SessionListener::onKernelRequest".{     "event": "kernel.request",     "listener": "Symfony\\Component\\HttpKernel\\EventListener\\SessionListener::onKernelRequest" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\LocaleListener::setDefaultLocale".{     "event": "kernel.request",     "listener": "Symfony\\Component\\HttpKernel\\EventListener\\LocaleListener::setDefaultLocale" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "FOS\RestBundle\EventListener\FormatListener::onKernelRequest".{     "event": "kernel.request",     "listener": "FOS\\RestBundle\\EventListener\\FormatListener::onKernelRequest" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\RouterListener::onKernelRequest".{     "event": "kernel.request",     "listener": "Symfony\\Component\\HttpKernel\\EventListener\\RouterListener::onKernelRequest" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\LocaleListener::onKernelRequest".{     "event": "kernel.request",     "listener": "Symfony\\Component\\HttpKernel\\EventListener\\LocaleListener::onKernelRequest" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\LocaleAwareListener::onKernelRequest".{     "event": "kernel.request",     "listener": "Symfony\\Component\\HttpKernel\\EventListener\\LocaleAwareListener::onKernelRequest" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "FOS\RestBundle\EventListener\BodyListener::onKernelRequest".{     "event": "kernel.request",     "listener": "FOS\\RestBundle\\EventListener\\BodyListener::onKernelRequest" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener::configureLogoutUrlGenerator".{     "event": "kernel.request",     "listener": "Symfony\\Bundle\\SecurityBundle\\Debug\\TraceableFirewallListener::configureLogoutUrlGenerator" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener::onKernelRequest".{     "event": "kernel.request",     "listener": "Symfony\\Bundle\\SecurityBundle\\Debug\\TraceableFirewallListener::onKernelRequest" }
CRITICAL19:00:43 | request | Uncaught PHP Exception App\Ldap\LdapDriverException: "An error occurred with the search operation." at LdapDriver.php line 72{     "exception": {} }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\DebugHandlersListener::configure".{     "event": "kernel.request",     "listener": "Symfony\\Component\\HttpKernel\\EventListener\\DebugHandlersListener::configure" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\ValidateRequestListener::onKernelRequest".{     "event": "kernel.request",     "listener": "Symfony\\Component\\HttpKernel\\EventListener\\ValidateRequestListener::onKernelRequest" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "Nelmio\CorsBundle\EventListener\CorsListener::onKernelRequest".{     "event": "kernel.request",     "listener": "Nelmio\\CorsBundle\\EventListener\\CorsListener::onKernelRequest" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "FOS\RestBundle\EventListener\ZoneMatcherListener::onKernelRequest".{     "event": "kernel.request",     "listener": "FOS\\RestBundle\\EventListener\\ZoneMatcherListener::onKernelRequest" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\SessionListener::onKernelRequest".{     "event": "kernel.request",     "listener": "Symfony\\Component\\HttpKernel\\EventListener\\SessionListener::onKernelRequest" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\LocaleListener::setDefaultLocale".{     "event": "kernel.request",     "listener": "Symfony\\Component\\HttpKernel\\EventListener\\LocaleListener::setDefaultLocale" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "FOS\RestBundle\EventListener\FormatListener::onKernelRequest".{     "event": "kernel.request",     "listener": "FOS\\RestBundle\\EventListener\\FormatListener::onKernelRequest" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\RouterListener::onKernelRequest".{     "event": "kernel.request",     "listener": "Symfony\\Component\\HttpKernel\\EventListener\\RouterListener::onKernelRequest" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\LocaleListener::onKernelRequest".{     "event": "kernel.request",     "listener": "Symfony\\Component\\HttpKernel\\EventListener\\LocaleListener::onKernelRequest" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "Symfony\Component\HttpKernel\EventListener\LocaleAwareListener::onKernelRequest".{     "event": "kernel.request",     "listener": "Symfony\\Component\\HttpKernel\\EventListener\\LocaleAwareListener::onKernelRequest" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "FOS\RestBundle\EventListener\BodyListener::onKernelRequest".{     "event": "kernel.request",     "listener": "FOS\\RestBundle\\EventListener\\BodyListener::onKernelRequest" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener::configureLogoutUrlGenerator".{     "event": "kernel.request",     "listener": "Symfony\\Bundle\\SecurityBundle\\Debug\\TraceableFirewallListener::configureLogoutUrlGenerator" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener::onKernelRequest".{     "event": "kernel.request",     "listener": "Symfony\\Bundle\\SecurityBundle\\Debug\\TraceableFirewallListener::onKernelRequest" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "App\EventSubscriber\RedirectToLocaleSubscriber::onKernelRequest".{     "event": "kernel.request",     "listener": "App\\EventSubscriber\\RedirectToLocaleSubscriber::onKernelRequest" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "App\EventSubscriber\WizardSubscriber::onKernelRequest".{     "event": "kernel.request",     "listener": "App\\EventSubscriber\\WizardSubscriber::onKernelRequest" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "Scheb\TwoFactorBundle\Security\TwoFactor\Event\TwoFactorFormListener::onKernelRequest".{     "event": "kernel.request",     "listener": "Scheb\\TwoFactorBundle\\Security\\TwoFactor\\Event\\TwoFactorFormListener::onKernelRequest" }
DEBUG19:00:43 | event | Notified event "kernel.request" to listener "App\EventSubscriber\UserEnvironmentSubscriber::prepareEnvironment".{     "event": "kernel.request",     "listener": "App\\EventSubscriber\\UserEnvironmentSubscriber::prepareEnvironment" }
DEBUG19:00:43 | event | Notified event "kernel.controller" to listener "App\EventSubscriber\UserProfileSubscriber::prepareUserProfile".{     "event": "kernel.controller",     "listener": "App\\EventSubscriber\\UserProfileSubscriber::prepareUserProfile" }
DEBUG19:00:43 | event | Notified event "kernel.controller" to listener "App\EventSubscriber\ThemeOptionsSubscriber::setThemeOptions".{     "event": "kernel.controller",     "listener": "App\\EventSubscriber\\ThemeOptionsSubscriber::setThemeOptions" }
DEBUG19:00:43 | event | Notified event "kernel.controller" to listener "FOS\RestBundle\EventListener\ParamFetcherListener::onKernelController".{     "event": "kernel.controller",     "listener": "FOS\\RestBundle\\EventListener\\ParamFetcherListener::onKernelController" }
DEBUG19:00:43 | event | Notified event "kernel.controller" to listener "Symfony\Bundle\FrameworkBundle\DataCollector\RouterDataCollector::onKernelController".{     "event": "kernel.controller",     "listener": "Symfony\\Bundle\\FrameworkBundle\\DataCollector\\RouterDataCollector::onKernelController" }
DEBUG19:00:43 | event | Notified event "kernel.controller" to listener "Symfony\Component\HttpKernel\DataCollector\RequestDataCollector::onKernelController".{     "event": "kernel.controller",     "listener": "Symfony\\Component\\HttpKernel\\DataCollector\\RequestDataCollector::onKernelController" }
DEBUG19:00:43 | event | Notified event "kernel.controller_arguments" to listener "Symfony\Component\Security\Http\EventListener\IsGrantedAttributeListener::onKernelControllerArguments".{     "event": "kernel.controller_arguments",     "listener": "Symfony\\Component\\Security\\Http\\EventListener\\IsGrantedAttributeListener::onKernelControllerArguments" }
DEBUG19:00:43 | event | Notified event "kernel.controller_arguments" to listener "Symfony\Component\HttpKernel\EventListener\CacheAttributeListener::onKernelControllerArguments".{     "event": "kernel.controller_arguments",     "listener": "Symfony\\Component\\HttpKernel\\EventListener\\CacheAttributeListener::onKernelControllerArguments" }
DEBUG19:00:43 | event | Notified event "kernel.controller_arguments" to listener "ContainerCQ6rYbD\RequestPayloadValueResolverGhost3590451::onKernelControllerArguments".{     "event": "kernel.controller_arguments",     "listener": "ContainerCQ6rYbD\\RequestPayloadValueResolverGhost3590451::onKernelControllerArguments" }
DEBUG19:00:43 | event | Notified event "kernel.controller_arguments" to listener "Symfony\Component\HttpKernel\EventListener\ErrorListener::onControllerArguments".{     "event": "kernel.controller_arguments",     "listener": "Symfony\\Component\\HttpKernel\\EventListener\\ErrorListener::onControllerArguments" }

</body>
</html>`

**Comment by kevinpapst** [comment]
Docker?

**Reply by stormframe** [reply]
no Docker

**Comment by stormframe** [comment]
ldap Search with Visual Studio Code - Ldap Plugin ist working, there is steht the Filter an also Bind DN + PW

```{
    "action": "ldap_search",
    "base_dn": "DC=firemanager,DC=local",
    "filter": "(&(&(objectClass=person))(uid=Michael))",
    "attributes": [
        "+",
        "*"
    ]
}```

**Comment by stormframe** [comment]
Any Ideas what can be testet?

**Comment by stormframe** [comment, marked as answer]
Hallo,
oft liegt das Problem tatsächlich an Kleinigkeiten :)
Es lag noch ein Fehler in der Config vor.... die BaseDN war nicht korrekt und der Fehler 500 war hier nicht sonderlich hilfreich :)
Läuft :D
Gruß
